blackbox/docs/release_notes/1.2.0.txt

```diff
@@ -0,0 +1,170 @@
+.. _version_1.2.0:
+
+=============
+Version 1.2.0
+=============
+
+Released on 2017/04/24.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.1.2` or higher before you upgrade to 1.2.0.
+
+   You cannot perform a :ref:`cluster_upgrade` to this version. Any upgrade to
+   this version will require a full cluster restart.
+
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Breaking Changes
+================
+
+ - To accommodate user-defined functions, some new reserved keywords have been
+   added to the CrateDB SQL dialect: ``RETURNS``, ``CALLED`, ``REPLACE``,
+   ``FUNCTION``, ``LANGUAGE``, ``INPUT``
+
+ - The ``enterprise.license`` setting is set to ``true`` by default.  
+   This enables the CrateDB `enterprise edition`_.  
+
+   Once you have evaluated the `enterprise edition`_ of CrateDB, you must
+   either buy an enterprise license, or disable this setting.  
+
+   If you disable this setting, CrateDB will run with the standard feature set.
+
+ - The default value of the ``node.max_local_storage_nodes`` node setting has
+   been changed to ``1`` to prevent running multiple nodes on the same data
+   path by default.  
+
+   Previous versions of CrateDB defaulted to allowing up to ``50`` nodes
+   running on the same data path. This was confusing where users accidentally
+   started multiple nodes and ended up thinking they have lost data because the
+   second node will start with an empty directory.  
+
+   Running multiple nodes on the same data path tends to be an exception,
+   so defaulting towards safer out-of-the-box settings is reasonable.
```

Comment:
"so defaulting towards safer out-of-the-box settings is reasonable" -> "so this is a safer default"